english/becomemarshmallow/becomemarshmallow.md

```diff
@@ -6,4 +6,4 @@ They return the favor and make a s'more out of you.
 
 RIP marshmallow you.
 
-You will be remembered!
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.